chips/nrf52/src/chip.rs

```diff
@@ -56,7 +57,21 @@ impl kernel::Chip for NRF52 {
                     match interrupt {
                         peripheral_interrupts::ECB => nrf5x::aes::AESECB.handle_interrupt(),
                         peripheral_interrupts::GPIOTE => nrf5x::gpio::PORT.handle_interrupt(),
-                        peripheral_interrupts::RADIO => ble_radio::RADIO.handle_interrupt(),
+                        peripheral_interrupts::RADIO => {
+                            match (
+                                ieee802154_radio::RADIO.is_enabled(),
+                                ble_radio::RADIO.is_enabled(),
+                            ) {
+                                (false, false) => (),
+                                (true, false) => ieee802154_radio::RADIO.handle_interrupt(),
+                                (false, true) => ble_radio::RADIO.handle_interrupt(),
+                                (true, true) => debug_assert!(
```

Comment:
should probably be `assert!` or `panic`, because we only use `--release`?

Reply:
This is taken from other code in the file, but I've changed it.